dev-resources/Documents/elrr_feature_branches.txt

```diff
@@ -0,0 +1,389 @@
+    e. git push --set-upstream elrrdatasync dev-pom-update-local-fixes-and-docs-1
+    f. git branch -a
+
+
+Commit to Remote Git dev-pom-update-local-fixes-and-docs-1 feature branch for elrrdatasync
+1. Open terminal 
+   a. cd C:\Users\phleven\git\elrrdatasync
+   b. git switch dev-pom-update-local-fixes-and-docs-1
+   c. git remote -v
+   d. mvn clean 
+   e. git status
+   f. git pull
+   g. git add <filename>
+   h. git commit -m "<message>"
+   i. git push
+   j. git log
+      1. Q (to quit)
+
+
+
+
+
+Create local feature branch dev-agg-update-local-fixes-and-docs-1 for elrraggregator
+ 1. Open terminal
+      a. cd C:\Users\phleven\git\elrraggregator
+      b. git switch agg-update
+      c. git switch -c dev-agg-update-local-fixes-and-docs-1
+      c. git branch -a
+
+
+Create new local shared dev feature branch dev-update-base for elrraggregator and push to GitHub
+ 1. Open terminal
+      a. cd C:\Users\phleven\git\elrraggregator
+      b. git switch agg-update
+      c. git switch -c dev-update-base
+      d. git push --set-upstream origin dev-update-base
+      e. git branch -a
+      f. git switch dev-agg-update-local-fixes-and-docs-1
+
+
+Create Pull Request from dev-agg-update-local-fixes-and-docs-1 to dev-update-base:
+1. URL = https://github.com/US-ELRR/elrraggregator
+
+2. Sign in
+   a. username = phleven
+   b. password = phil55.cindy61
+   
+3. Pull requests --> New pull request
+   a. base = dev-update-base, compare = dev-agg-update-local-fixes-and-docs-1
+   b. Create pull request
+      1. Reviewers
+         a. cliffcaseyyet
+      2. Asignees
+         a. phleven
+      3. Description
+         a. <description>
+   c. Create pull request
+
+
+Commit changes to local dev-agg-update-local-fixes-and-docs-1 feature branch and push to remote for elrraggregator
+ 1. Open terminal
+      a. cd C:\Users\phleven\git\elrraggregator
+      b. git switch dev-agg-update-local-fixes-and-docs-1
+      c. git status
+      d. git add <fileName>
+      e. git commit -m "<Description>"
+      f. git push
+      g. git log
+         1. Q (to quit)
+
+
+Push changes to remote branch for elrraggregator
+ 1. Open terminal
+    a. cd C:\Users\phleven\git\elrraggregator
+    b. git remote add elrraggregator https://github.com/US-ELRR/elrraggregator.git
+    c. git remote -v
+    d. git push --set-upstream elrraggregator dev-agg-update-local-fixes-and-docs-1
+    e. git branch -a
+
+
+Push changes and create new remote dev-agg-update-local-fixes-and-docs-1 feature branch for elrraggregator
+ 1. Open terminal
+    a. cd C:\Users\phleven\git\elrraggregator
+    b. git switch dev-svc-update-local-fixes-and-docs-1
+    c. git remote add elrraggregator https://github.com/US-ELRR/elrraggregator.git
+    d. git remote -v
+    e. git push --set-upstream elrraggregator dev-agg-update-local-fixes-and-docs-1
+    f. git branch -a
+
+
+Commit to Remote Git dev-agg-update-local-fixes-and-docs-1 feature branch for elrraggregator
+1. Open terminal 
```

Comment:
Some of these aren't even part of the project.

Please pare this documentation down solely to instructions for RUNNING this project, not a git guide, with no references to your filesystem or branches.

The other stuff you are welcome to hold onto but it doesn't make sense in this repo.